sdk/tables/azure-data-tables/azure/data/tables/_table_client.py

```diff
@@ -250,29 +252,27 @@ def create_entity(
             entity,  # type: Union[TableEntity, dict[str,str]]
             **kwargs  # type: Any
     ):
-        # type: (...) -> TableEntity
+        # type: (...) -> dict[str,str]
```

Comment:
should be capital d 'Dict'. As a rule of thumb, mypy typing is upper case (ie `Dict`, `List`), while sphinx (what we put in the docstrings), is lowercase